lib/jellyfish/component/hls.ex

```diff
@@ -14,19 +14,40 @@ defmodule Jellyfish.Component.HLS do
   alias Membrane.RTC.Engine.Endpoint.HLS.{CompositorConfig, HLSConfig, MixerConfig}
   alias Membrane.Time
 
+  @cleanup_after Time.seconds(60)
   @segment_duration Time.seconds(6)
   @partial_segment_duration Time.milliseconds(1_100)
-
   @type metadata :: %{
+          optional(:target_window_duration) => pos_integer(),
           playable: boolean(),
-          low_latency: boolean()
+          low_latency: boolean(),
+          persistent: boolean()
         }
 
   @impl true
   def config(options) do
     with {:ok, valid_opts} <- OpenApiSpex.cast_value(options, Options.schema()) do
       low_latency? = valid_opts.lowLatency
-      hls_config = create_hls_config(options.room_id, low_latency?: low_latency?)
+      target_window_duration = valid_opts.targetWindowDuration
```

Comment:
CamelCase in the elixir code looks strange. Is this because in OpenApi camelCase is used?

Reply:
Yup, that remains the only place where we _have_ to use camelcase (if we want to keep doing things this way)